.github/workflows/pull_request_verification.yml

```diff
@@ -5,27 +5,23 @@ env:
   COMMITS_COUNT: ${{ github.event.pull_request.commits }}
 
 jobs:
-  branch-name:
+  assert-pull-request-metadata:
```

Comment:
What is the purpose of reducing the number of jobs? Just to be more readable or has it any performance advantages?

Reply:
Main purpose is to keep it simple. I don't see any benefit of keeping them as two separate jobs, since they're light-weight and fast checks. Most of the time is consumed by the setup of the environment.